### ISSUE TITLE
Swipe To Slide

### ISSUE THREAD

**Issue by kestraly** [magiccart/magento-2-product-slider#5]
Firstly, it's a shame you put this behind a paywall. It was very nice extension to have.

Secondly, you should include swipeToSlide as an option to improve mobile usability.

**Comment by magiccart** [comment]
Thanks for the report. You still can install it free via composer. We don't have expenses for maintenance so we will provide a version upgrade lifetime with a little cost. If you can maintain and contribute to this repository will be good.

**Comment by kestraly** [comment]
Yeah, absolutely. I think there's lots that should be tapped into for the Slick Slider library. 

Things like number to scroll, swipetoslide, responsive settings in general. I find slick out of the box is very clunky, however getting the slider to behave without glitching in Magento requires a lot of tweaking.

**Comment by kestraly** [comment]
@magiccart - On 2.4.5 did you come across a problem that the product collection is the same if you have more than 1 slider on the home page? Have 3 sliders, each one is for featured for a different category. Since 2.4.5 the product collection is the same for every slide.

For Future Me: The following fixed the problem https://github.com/magento/magento2/commit/a9170022cce29993e586ffaa6f4c1cc357ee3b11.diff
